arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1550,7 +1575,7 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition."
```

Comment:
Resource model is for Proxy(untracked) resources like Queue, Topic and Subscription. TrackedResource is for Tracked Resource like Namespace. We don't need Location and Tags in Proxy resource, so made two separate models. Suggested by Deepak Shankargouda on Relay review email thread to resolve the Tracked resource - Patch operations errors.